src/host.cpp

```diff
@@ -921,7 +921,11 @@ namespace rhost {
             do_r_callback(true);
         }
 
-        extern "C" int R_ReadConsole(const char* prompt, char* buf, int len, int addToHistory) {
+#ifdef _WIN32
+        extern "C" int R_ReadConsole(const char* prompt, char * buf, int len, int addToHistory) {
```

Comment:
Use a typedef for `buf` instead, and do ifdef around that - it'll be a bit clearer.